colin-api/src/colin_api/models/filing.py

```diff
@@ -1116,7 +1131,8 @@ def add_involuntary_dissolution_event(cls, con, corp_num, filing_body) -> int:
     def add_filing(cls, con, filing: Filing) -> int:
         """Add new filing to COLIN tables."""
         try:
-            if filing.filing_type not in ['alteration', 'amalgamationApplication', 'annualReport', 'changeOfAddress',
+            if filing.filing_type not in ['agmLocationChange', 'alteration', 'amalgamationApplication',
+                                          'annualReport', 'changeOfAddress',
```

Comment:
Is sync needed for AGM Extension? Would it make sense to do that one at the same time?

Reply:
OK. Resolving this conversation. Merge when ready :)